### DISCUSSION TITLE
Account Verification - process is awful, I can't recommend GitHub anymore

### DISCUSSION THREAD

**Discussion by mseryn** [community/community#57833] [GitHub Education]
### Hi everyone,

The new process for verifying an education account is horrible. 

First, a disclaimer. I recognize forum moderators and support persons don't make platform decisions. I know the GitHub person(s) responding to this forum likely didn't decide to change how GitHub for Education works. I'm writing this in the hopes that even just a little feedback can be passed up to the ears of someone who makes said platform decisions. Because this is...bad.

So, to the decision-makers:

Unless I'm missing something, which is always possible, it's awful to use the new system. I dug around all over for different options to verify my education account, and I found no solutions to the awfulness. If there is a way to use the verification system better, **_please improve the site documentation_**. I don't have time to fight with GitHub for hours just to get my account to work, not when there are competing platforms that make it beyond-easy to get an education account.

Observations:

- The automated system only takes camera images (not file uploads?). Am I supposed to print my enrollment verification and shove it in front of my camera? Open it in a browser and photograph my screen with my phone? This is idiotic and inefficient, especially given this is a technology platform.
- You can't contact support for GitHub for Education - it tells you "we no longer use humans to verify accounts". Trying to go through standard GitHub support gets you a "closing ticket, please reach out to GitHub for Education". So if I have an issue verifying my account, there's no recourse. None.
- Having to post to a discussion platform to fix personal account issues is a weird, nonstandard, and privacy-ignorant mechanism.
- All over this discussion platform, I'm seeing issues about account verification. This is not just me. Seriously, guys, you need to either fix these issues or better-document the ways to use the verification system better.
- I keep seeing "issue was fixed when I used the site on my phone", which is at least something, except... this is GitHub. Many of us are programmers. We use computers - and phones, of course, but come on - fix your site in the browser.

I get it. Verifying accounts is hard. Putting in (paid) human effort for (unpaid) student accounts is also costly. But it's not a secret that students take familiar platforms with them into the professional world. If GitHub isn't convenient to use (for free) during their education, they'll just use another platform, and they'll take _that_ with them into professional and personal projects. This isn't a trade secret or anything, it's well-known.

I'm just one person, and I know that - but I'm almost definitely not alone. So here's the loud opinion from this one person who is sad to see her GitHub days at a close: this system has decayed, and _I won't use or recommend GitHub for Education anymore_. I'm about to defend and am planning to go directly into research and professorship. I'm one person, but I do educate and used to push dozens to hundreds of students per year to use GitHub for Education (and beyond). I'm also going to write a simple PSA for others - labmates, other instructors, my advisors, other researchers, and anyone else relevant - and recommend that they go through this new process to see how bad it is, just so they can come to their own mind about if they should subject their students or labmates to this. Because I absolutely will not. 

You guys need to fix this system.  It's really, really bad, and people in tech do not have patience for a rotting platform when others readily exist.

And to anyone who does not make decisions about platform mechanisms, thanks for your time, and I hope you don't get as many headaches dealing with this system as I did trying to use it.

**Comment by pmcgeebcit** [comment]
Thank you for posting this. That has been my experience as well. I submitted a ticket on Jue 23 about this issue and still I have not heard back one month later. All I have requested in my ticket is to complete the academic account verification process (it is actually a renewal). Hopefully GitHub can fix this soon since GitHub for Education has some excellent services.

**Comment by xdlai** [comment]
The shitty verification process only pushes people to use other platforms. I can not image who brings up such a stupid idea.

**Comment by kshannon** [comment]
100% my experience as well. Wasted time and effort.

**Comment by huongtr** [comment]
100% they even didn't test it before deploying. shitty

**Comment by AtifAhmadSiddiqui** [comment]
Who came up with this verification process? 15 Dice answers to create an account in Guthub. Sheer waste of time. 
![Github_Verifications](https://github.com/community/community/assets/10746935/7851d052-83df-4556-bdff-a1577a7b9213)

**Reply by smileservices** [reply]
I know, gives me headache. It's easier to spin up your own gitlab machine on a cloud vm.

**Comment by Aary23S** [comment]
I am unable to set my location to correct the options always showing  me the wrong choice of location of my university. Please help

**Comment by zhangzhize** [comment]
Same, today. Can't Believe it.

**Comment by danielj23** [comment]
This is insanity and when it breaks, you are forced to start over and do 20 more?

Crazy and terrible.  I can't even complete it so... I guess I just give up?   

Who made this? 

 
![2024-03-29 13_43_52-Join GitHub · GitHub — Mozilla Firefox](https://github.com/community/community/assets/725527/2dbdf2d6-817c-4632-8387-4b2f7172cbe8)

**Comment by himanish-k** [comment]
We're supposed to do 20 of these? And if it's wrong, you start over. That's crazy. I guess it's for protection against automated account creation by bots but this is horrible UX for actual users.

Comically, you can't comment on this thread unless you already have an account.

**Comment by ivanjrt** [comment]
is a massive discouragement on creating an account, 20 tricky "only they know the answer" screenshots on the other option, you have to respond audio then is even worse.

I get it if it was just 5 attempts

here's an example of this failure:

![image](https://github.com/community/community/assets/44326428/e5c62e72-4d1a-4492-95a8-18af405b34b2)
![image](https://github.com/community/community/assets/44326428/2f7b2435-30e6-4456-9fa7-87bee99fef3a)
![image](https://github.com/community/community/assets/44326428/8c1dd1c2-25c8-4365-b53e-578607277f94)
see how confusing can this be, who did this?, chatGpt 😆

**Comment by minhnhat2807** [comment]
Crazy github

**Comment by flampoo** [comment]
Clearly I'm not the only one who had a problem with this, but wanted to comment as well to show how absurd this process is. 

Had to go through the verification process 4 separate times. Each time I had a choice similar to @ivanjrt. 2-3 options identical to the naked eye. One such example included below. 

![github 15 1](https://github.com/community/community/assets/168282944/d71c4af0-a9bd-401f-a8cc-aaf3ef2181ca)
![github 15 2](https://github.com/community/community/assets/168282944/9e3fcc2c-4964-4560-8d84-7b9ab16a87b1)
![github 15 3](https://github.com/community/community/assets/168282944/a67481f4-5065-4805-b4e9-cf56e010ebc3)


I'm guessing this is training an AI or something, but, Christ, maybe it should be training us at this point if it can distinguish between those three things.

**Comment by khaduykha** [comment]
Yes, I totally agree, this is a stupid idea.

**Comment by aftabmj** [comment]
I wanted to create an online repo for my new startup. I have been using github for 7+ years.  After being unable to solve the visual and audio steps for 30+ minutes, I am switching to some other provider. Seriously who comes up with these ideas ..how does it get approved ??

**Comment by AnushaPatelHS** [comment]
This is really annoying way of verification. Please find other way were it can be human friendly if you want to verify if its human. 
Other than this everything is just perfect :) Please check on this verification process as this is time consuming and sometimes its just not possible to have so much of time like during workshop.

**Comment by vxgu86** [comment]
i have to say some in here, I have tried verified the student profit for so many time that I cannot remember
is there really someone pass the verification?
**We need a little more proof to verify your academic status.**  even I post everything on it.

**Reply by vxgu86** [reply]
so so so  annoying

**Comment by nmaddp1995** [comment]
Oh my god, After 10 years of using GitHub, I'm trying to create another account to test things.
And Surprisingly I cannot pass the damn sh!t account verification process.
I'm not sure if there is an actual person who can create an account after all these sh!t things
Please GitHub owner, do something to save your product

**Comment by jakbin** [comment]
Yes github signup really sucks.

**Reply by danielj23** [reply]
Can't believe I am still receiving notifications that people have this issue.  Microsoft at their best.

**Comment by Shifath472533** [comment]
This is very true. Absolutely rubbish system. The system is dumb.

**Comment by marklaw59** [comment]
Very annoyed with the new verification process.  It won't even use the back camera on the phone so I can see what I'm imaging.

**Comment by u15767** [comment]
Same here. I don't know, why I now even have to create an organization? That is complete nonsense.

**Comment by AVandana-1806** [comment]
Yes I agree one of the worst experience in creating the new account and very horrible I almost wasted 2 hours in my life for this crap and still unable to create the account.

**Comment by LouiseDerizhy** [comment]
what is this? my eye and brain hurt. i cant verify my account
<img width="466" height="499" alt="image" src="https://github.com/user-attachments/assets/ab455c67-571b-4a70-814a-5484df28a06e" />

**Comment by Yassine-ala** [comment]
I can't create an account on github due to the genius vérification process. 
Context : I need to create an account with my professional mail to activate a copilot license. 
I've failed twice the process, once with the logo cups challenge (browse 10 possible outcomes, 10 times) apparently one logo didnt match the blurry logos on the picture, go wonder, also might have been a failure at the 3rd time so the game didn't mind torturing me for the rest of the 7 tries for nothing.
Second audio test had a toll on my mental health, i had to listen to animal noises, bees, cats and what i think is zombie noises in between, 10 sets of 3 animal noises audios, failed as well, i don't know why, maybe confusing zombie noises with cat ones, only the dev knows.
Anyone get these right, i'm doubting my human capabilities at this point.

**Comment by Yassine-ala** [comment]
After 2 coffees and with coworker help, we managed to pass the cups test, good luck all.

**Comment by coolg123** [comment]
I have raised one support ticket since I am always getting rejected even though I submitting original documents but still could you anyone tell me how much time it will take or do I need comment anywhere to make them notice that ticket

**Comment by josephwatson5237-web** [comment]
It’s frustrating to see such a vital academic tool become harder to access, especially when smooth verification should feel as streamlined as managing workflows with [LemTask](https://lemtask.com/).
Hopefully the feedback here reaches the right team, because platforms built for developers should prioritize clarity, usability, and real support.

**Comment by ak0190** [comment]
Just tried to get a friend setup for github. Failed with the verification. Going to move to gitlab now. Stupid Github

**Comment by uk974q** [comment]
<img width="1145" height="895" alt="image" src="https://github.com/user-attachments/assets/ced01c81-e92d-4d8f-b0ba-8ccca304d6a4" />

Seriously!! 10 ??
